Better-sqlite3 not building in with Electron 11

Better-sqlite3 doesn't seem to want to build in Electron 11.  When starting the electron app for the first time (when it prepares native modules), it produces this:
```

  ACTION deps_sqlite3_gyp_locate_sqlite3_target_extract_sqlite3 b857c92884e9598d609f6be182a2595df7a8e00f.intermediate
⠦ Preparing native dependencies: 0 / 1  TOUCH Release/obj.target/deps/locate_sqlite3.stamp
  CC(target) Release/obj.target/sqlite3/gen/sqlite3/sqlite3.o
⠹ Preparing native dependencies: 0 / 1  LIBTOOL-STATIC Release/sqlite3.a
⠦ Preparing native dependencies: 0 / 1  CXX(target) Release/obj.target/better_sqlite3/src/better_sqlite3.o
⠏ Preparing native dependencies: 0 / 1In file included from ../src/better_sqlite3.cpp:4:
In file included from ./src/better_sqlite3.lzz:10:
In file included from /Users/ben/Documents/GitHub/Assessment-Disaggregation/node_modules/better-sqlite3/11.0.1/include/node/node.h:67:
In file included from /Users/ben/Documents/GitHub/Assessment-Disaggregation/node_modules/better-sqlite3/11.0.1/include/node/v8.h:30:
/Users/ben/Documents/GitHub/Assessment-Disaggregation/node_modules/better-sqlite3/11.0.1/include/node/v8-internal.h:418:38: error: 
      no template named 'remove_cv_t' in namespace 'std'; did you mean
      'remove_cv'?
            !std::is_same<Data, std::remove_cv_t<T>>::value>::Perform(data);
                                ~~~~~^~~~~~~~~~~
                                     remove_cv
/Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../include/c++/v1/type_traits:697:50: note: 
      'remove_cv' declared here
template <class _Tp> struct _LIBCPP_TEMPLATE_VIS remove_cv
                                                 ^
⠙ Preparing native dependencies: 0 / 11 error generated.
make: *** [Release/obj.target/better_sqlite3/src/better_sqlite3.o] Error 1
rm b857c92884e9598d609f6be182a2595df7a8e00f.intermediate
✖ Preparing native dependencies: 0 / 1

An unhandled error has occurred inside Forge:
node-gyp failed to rebuild '/Users/ben/Documents/GitHub/Assessment-Disaggregation/node_modules/better-sqlite3'.
Error: `make` failed with exit code: 2


Error: node-gyp failed to rebuild '/Users/ben/Documents/GitHub/Assessment-Disaggregation/node_modules/better-sqlite3'.
Error: `make` failed with exit code: 2

```
Thanks in advance!

Note:
Intel Mac, running Big Sur

--- a/binding.gyp
+++ b/binding.gyp
@@ -9,9 +9,9 @@
       'target_name': 'better_sqlite3',
       'dependencies': ['deps/sqlite3.gyp:sqlite3'],
       'sources': ['src/better_sqlite3.cpp'],
-      'cflags': ['-std=c++11'],
+      'cflags': ['-std=c++14'],
       'xcode_settings': {
-        'OTHER_CPLUSPLUSFLAGS': ['-std=c++11', '-stdlib=libc++'],
+        'OTHER_CPLUSPLUSFLAGS': ['-std=c++14', '-stdlib=libc++'],
       },
     },
     {

